client/client.go

```diff
@@ -321,6 +321,12 @@ func (c *Client) RefreshFeed(feedID int64) error {
 	return err
 }
 
+// MarkFeedAsRead mark all entries from a feed as read since last check.
+func (c *Client) MarkFeedAsRead(feedID int64) error {
+	_, err := c.request.Get(fmt.Sprintf("/v1/feeds/%d/mark_all_as_read", feedID))
```

Comment:
I don't see any change regarding the API, this endpoint doesn't exists? If you are planning to add it later, the method should be a PUT or POST, not GET. Otherwise, just remove this function for now.

Reply:
I've removed and will add in another patch with the updated documentation